site/content/docs/main/guides/structured-logs.md

```diff
@@ -1,86 +0,0 @@
----
```

Comment:
This was replaced by `config/access-logging.md` awhile back so dropping from the guides section.